static/css/style.css

```diff
@@ -410,81 +246,26 @@ ol {
   margin-right: auto;
 }
 
-@media screen and (max-width: 1030px) {
-  .sidebar-left {
-    position: fixed;
-    background: #000000;
-    top: 0;
-    width: 100%;
-    height: 6em;
-    z-index: 2;
-  }
-
-  .sidebar-right {
-    display: none;
-  }
-
-  .header-title {
-    display: flex;
-    margin: 0;
-    padding: 0;
-  }
-
-  .header-title div {
-    flex: 1;
-  }
-
-  .header-title .category-list li {
-    display: inline-block;
-  }
-
-  .header-title .category-list li a,
-  .header-title .tags-list li a {
-    font-size: 24px;
-  }
-
-  .header-title .title {
-    display: none;
-  }
-
-  .header-title .avatar {
-    margin: 0;
-    width: 4em;
-    height: 4em;
-    border-width: 2px;
-  }
-
-  .header-title .tagline {
-    display: none;
-  }
-
-  .tags-list {
-    display: none;
-  }
-
-  .title-container, .tagline-container, .social-container {
-    display: none;
-  }
-
-  .content {
-    margin: 0;
-    margin-top: 7em;
-    padding: 1em;
-    min-width: 712px;
-  }
-
-  article img {
-    width: 100%;
-  }
-}
-
-
-/* On screens that are 600px or less, set the background color to olive */
-
-/*
-@media screen and (max-width: 600px) {
-  body {
-    background-color: olive;
-  }
-}
-*/
\ No newline at end of file
+iframe {
+  width: 100%;
+}
+
+.col-xs-3 {
+    max-width: 20%!important;
+    flex-basis: 20%!important;
```

Comment:
remove `!important`, please do not overwrite original library tags, there should be a better way.